CI failed. It looks like this exposed a possible bug:
```
fvolumesum.cpp: In function ‘void main_main()’:
fvolumesum.cpp:185:56: error: ‘vol’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
                                     lsum += fab(i,j,k) * vol;
                                             ~~~~~~~~~~~^~~~~
fvolumesum.cpp:142:56: error: ‘vol’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
                                     lsum += fab(i,j,k) * vol;
                                             ~~~~~~~~~~~^~~~~
```

--- a/Tools/Plotfile/fvolumesum.cpp
+++ b/Tools/Plotfile/fvolumesum.cpp
@@ -120,7 +120,7 @@ void main_main()
                                                         problo[2]+static_cast<Real>(k+0.5)*dx[2])};
 
                                     // compute the volume
-                                    Real vol;
+                                    Real vol = NAN;
                                     if (coord == 0) {
                                         // Cartesian
                                         vol = 1.0_rt;

--- a/Tools/Plotfile/fvolumesum.cpp
+++ b/Tools/Plotfile/fvolumesum.cpp
@@ -120,7 +120,7 @@ void main_main()
                                                         problo[2]+static_cast<Real>(k+0.5)*dx[2])};
 
                                     // compute the volume
-                                    Real vol = NAN;
+                                    Real vol = std::numeric_limits<Real>::quiet_NaN();
                                     if (coord == 0) {
                                         // Cartesian
                                         vol = 1.0_rt;
@@ -163,7 +163,7 @@ void main_main()
                                                     problo[2]+static_cast<Real>(k+0.5)*dx[2])};
 
                                     // compute the volume
-                                    Real vol = NAN;
+                                    Real vol = std::numeric_limits<Real>::quiet_NaN();
                                     if (coord == 0) {
                                         // Cartesian
                                         vol = 1.0_rt;

--- a/Tools/Plotfile/fvolumesum.cpp
+++ b/Tools/Plotfile/fvolumesum.cpp
@@ -163,7 +163,7 @@ void main_main()
                                                     problo[2]+static_cast<Real>(k+0.5)*dx[2])};
 
                                     // compute the volume
-                                    Real vol;
+                                    Real vol = NAN;
                                     if (coord == 0) {
                                         // Cartesian
                                         vol = 1.0_rt;

